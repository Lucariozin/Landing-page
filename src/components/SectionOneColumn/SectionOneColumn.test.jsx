import { render, screen } from '@testing-library/react';
import { SectionOneColumn } from '.';
import Mock from './mock';

describe('<SectionOneColumn />', () => {
  it('should render <SectionTwoColumns /> correctly', () => {
    render(<SectionOneColumn {...Mock} />);

    expect(
      screen.getByRole('heading', {
        name: /News coverage and some surprises/i,
      }),
    );

    expect(
      screen.getAllByText(/The release of Apple Silicon-based/i),
    ).toHaveLength(3);
  });

  it('should to match snapshot', () => {
    const { container } = render(<SectionOneColumn {...Mock} />);
    expect(container).toMatchSnapshot();
  });
});
