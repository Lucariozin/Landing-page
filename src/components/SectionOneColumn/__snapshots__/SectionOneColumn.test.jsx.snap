// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionOneColumn /> should to match snapshot 1`] = `
.c2 {
  margin: 0 auto;
  margin-bottom: 2rem;
  max-width: 65rem;
  text-align: center;
}

.c3 {
  margin-bottom: 5rem;
}

.c5 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 2.5rem;
}

.c0 {
  width: 100%;
  min-height: 100vh;
  padding: 1rem;
  padding-top: 8rem;
  background-color: var(--white);
  color: var(--primary-color);
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
}

.c4 {
  color: var(--primary-color);
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c6 {
  color: var(--primary-color);
  font-style: normal;
  font-size: 2rem;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4rem;
  }
}

<div>
  <div
    class="c0"
    id=""
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h2
            class="c4"
          >
            News coverage and some surprises
          </h2>
        </div>
        <div
          class="c5"
        >
          <p
            class="c6"
          >
            The release of Apple Silicon-based Macs at the end of last year generated a flurry of news coverage and some surprises at the machine’s performance. This post details some background information on the experience of porting Firefox to run natively on these CPUs.
          </p>
          <p
            class="c6"
          >
            The release of Apple Silicon-based Macs at the end of last year generated a flurry of news coverage and some surprises at the machine’s performance. This post details some background information on the experience of porting Firefox to run natively on these CPUs.
          </p>
          <p
            class="c6"
          >
            The release of Apple Silicon-based Macs at the end of last year generated a flurry of news coverage and some surprises at the machine’s performance. This post details some background information on the experience of porting Firefox to run natively on these CPUs.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
